packages/destination-actions/src/destinations/hyperengage/validateInput.ts

```
@@ -67,11 +47,20 @@ export const validateInput = (
 
   // Validate user properties
   if (event_type === 'user_identify') {
-    properties.traits = {
-      email: input?.email,
-      name: input?.name,
-      created_at: input?.created_at,
-      ...properties.traits
+    if (input?.name) {
+      properties.traits = {
+        email: input?.email,
+        name: input?.name,
+        created_at: input?.created_at,
+        ...properties.traits
+      }
+    } else if (input?.first_name || input?.last_name) {
+      properties.traits = {
+        email: input?.email,
+        name: `${input?.first_name} ${input?.last_name}}`,
+        created_at: input?.created_at,
+        ...properties.traits
+      }
```

Should we throw an error if input.name and input.first_name and input.last_name are all empty?

---

Yes, I think throwing an error for mandatory field missing should be the way to go here. Lemme see what I can do.

---

Hey @joe-ayoub-segment , I've resolved the default path issues and added an error incase none of the first_name and last_name or name fields are provided. Sorry for the inconvenience and thank you so much for your time